src/marketing/price-rules-catalog-create.md

```diff
@@ -55,133 +55,135 @@ Follow these instructions to apply a discount to specific products whenever a se
+   _Condition Line 2, Part 3_
 
    The selected item appears in the statement to complete the condition.
 
-      If <span class="underlined">ALL</span> of these conditions are <span class="underlined">TRUE</span>: </ br>Attribute Set <span class="underlined">is</span> Default
+   If **ALL** of these conditions are **TRUE**: <br/> Attribute Set **is Default**
 
-1. To add another condition line to the statement, tap the **Add** (![]({% link images/images/btn-add-grn.png %})) button, and choose one of the following:
+1. To add another condition line to the statement, click the **Add** (![Add icon]({% link images/images/btn-add-grn.png %})) icon and choose one of the following:
 
-   * Conditions Combination
-   * Product Attribute
+   - `Conditions Combination`
+   - `Product Attribute`
 
    Then, repeat the process until all desired conditions are complete.
 
-   If at any time you want to delete part of the condition statement, tap the **Delete** (![]({% link images/images/btn-del-red.png %})) button at the end of the line.
+   If at any time you want to delete part of the condition statement, click the **Delete** (![Delete icon]({% link images/images/btn-del-red.png %})) icon at the end of the line.
 
 ## Step 3: Define the Actions
 
-1. Expand ![]({% link images/images/btn-expand.png %})the **Actions** section, and do the following:
+1. Expand ![Expansion selector]({% link images/images/btn-expand.png %})the **Actions** section and do the following:
 
-   ![]({% link images/images/price-rule-catalog-actions.png %}){: .zoom}
-   *Actions*
+   ![Catalog price rule - actions]({% link images/images/price-rule-catalog-actions.png %}){: .zoom}
+   _Actions_
 
 1. Under **Pricing Structure Rules**, set **Apply** to one of the following:
 
-   |||
-   |--- |--- |
    |Apply as percentage of original|Discounts item by subtracting a percentage of the regular* price. For example: Enter 10 in Discount Amount for an final** price that is marked down 10% from the regular price.|
    |Apply as fixed amount|Discounts item by subtracting a fixed amount from the regular price. For example: Enter 10 in Discount Amount for a final price that is $10 less than the regular price.|
    |Adjust final price to this percentage|Adjusts the final price by a percentage of the regular price. For example: Enter 50 in Discount Amount for a final price that is marked down 50% from the regular price.|
-   |Adjust final price to discount value|Sets the final price to a fixed, discounted amount. For example:Enter 20 in Discount Amount for a final  price of $20.00.|
+   |Adjust final price to discount value|Sets the final price to a fixed, discounted amount. For example: Enter 20 in Discount Amount for a final price of $20.00.|
    ||Regular price refers to the base product price without any advanced pricing  (special/tier/group) or promotional discounts. Final price refers to the discounted price that appears in the shopping cart.|
```

Comment:
this row doesn't have a field name

Reply:
I think those are from the * and ** items. It only refers to items in the one cell, so I think I will move this there. I hate footnotes, so happy to be rid of that anyway.